composer.json

```diff
@@ -92,5 +93,11 @@
       ]
     }
   },
-  "minimum-stability": "beta"
+  "minimum-stability": "beta",
+  "repositories": [
```

Comment:
i've added package to packagist, so you can use package name without repository add